bmap-rs/src/main.rs

```diff
@@ -1,6 +1,7 @@
 use anyhow::{anyhow, bail, Context, Result};
 use bmap::{Bmap, Discarder, SeekForward};
 use flate2::read::GzDecoder;
+use hyper::Uri;
```

Comment:
if you are using the reqwest lib to handle the actual downloading, can you not use `reqwest::Url` instead ?

Reply:
I think we don't need to use reqwest for the moment. Unless we have to implement the range-request download